R/log_lik.R

```diff
@@ -734,6 +734,17 @@ log_lik_zero_inflated_binomial <- function(i, prep) {
   log_lik_weight(out, i = i, prep = prep)
 }
 
+log_lik_zero_inflated_beta_binomial <- function(i, prep) {
+  trials <- prep$data$trials[i] 
+  mu <- get_dpar(prep, "mu", i) 
+  phi <- get_dpar(prep, "phi", i) 
+  zi <- get_dpar(prep, "zi", i)
+  args <- list(size = trials, mu = mu, phi = phi, zi)
```

Comment:
`zi` is unanamed here for some reason. Is this intentional? you could also use `nlist()` here to avoid naming mu, phi, and zi.

Reply:
Named now, didn’t realise `nlist` could take named args as well, cool helper